endtoend-testing/src/test/java/de/jplag/endtoend/LanguageHelper.java

```diff
@@ -0,0 +1,45 @@
+package de.jplag.endtoend;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import de.jplag.Language;
+
+/**
+ * Helper class to load a language based on its identifier. In order for languages to be loaded, their language module
+ * dependency has to be added to the pom.xml file and a matching enum case has to be added.
+ */
+public enum LanguageHelper {
+    /** The @{de.jplag.java.Language Java language} */
+    JAVA("java");
+
```

Comment:
I think E2E Tests can rely on the language loader class of the CLI. This class should also be tested in E2E (from my point of view)

Reply:
Fine from my perspective, just thought that it might not be the best solution to have a dependency on the CLI in the E2E tests. Maybe we ask @tsaglam for a deciding opinion.

Reply:
I think both solutions are fine. Including the CLI will ensure that more code is covered and is probably better for future extensions though. Maybe let us do it this way. No strong opinion though.